cookbook/import_export/product-import-from-xml-file.rst

```diff
@@ -93,6 +104,21 @@ And we introduce the following extension to load the services files in configura
    :language: php
    :linenos:
 
+Translate Job and Step labels in the UI
+---------------------------------------
+
+Behind the scene, the service ``Pim\Bundle\ImportExportBundle\JobLabel\TranslatedLabelProvider`` provides translated Job and Step labels to be used in the UI.
+
+This service uses following conventions:
+ - for a job label, given a $jobName, "batch_jobs.$jobName.label"
```

Comment:
I think you should use `%variable%` instead of `$variable` but this is just a preference